src/borg/testsuite/archiver/argparsing.py

```diff
@@ -11,6 +12,23 @@ def test_bad_filters(self):
         self.cmd(f"--repo={self.repository_location}", "create", "test", "input")
         self.cmd(f"--repo={self.repository_location}", "delete", "--first", "1", "--last", "1", fork=True, exit_code=2)
 
+    def test_highlander(self):
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        self.cmd(f"--repo={self.repository_location}", "create", "--comment", "comment 1", "test-1", src_dir)
+        error_msg = "There can be only one"
+        # Default permission is 077
```

Comment:
you mean "default value of --umask"?

Reply:
also, it is 0077 (according to the docs). the initial 0 means "octal", followed by 3 octal digits for the value.